cypress/e2e/tests/migration/applicationinventory/assessment/create.test.ts

```diff
@@ -102,6 +103,16 @@ describe(["@tier2"], "Application validations", () => {
         inputText(applicationNameInput, data.getFullName());
         cy.get(commonView.submitButton).should("not.be.disabled");
 
+        // Contributors Validation, Polarion TC 331
+        inputText(applicationContributorsInput, stakeHoldersList[0].name);
+        cy.get("button").contains(stakeHoldersList[0].name).click();
+        inputText(applicationContributorsInput, stakeHoldersList[1].name);
+        cy.get("button").contains(stakeHoldersList[1].name).click();
+        cy.get(applicationContributorsInput)
+            .parent()
+            .should("contain", stakeHoldersList[0].name)
```

Comment:
Let's also remove one contributor here and see if it gets updated .

Reply:
Done